front/lib/tracking/customerio/server.ts

```diff
@@ -0,0 +1,175 @@
+import type {
+  LightWorkspaceType,
+  MembershipRoleType,
+  UserType,
+  UserTypeWithWorkspaces,
+} from "@dust-tt/types";
+import * as _ from "lodash";
+
+import { subscriptionForWorkspace } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { MembershipResource } from "@app/lib/resources/membership_resource";
+import { renderLightWorkspaceType } from "@app/lib/workspace";
+
+const {
```

Comment:
We have moved from `process.env` to `import config from "@app/lib/api/config";`

Can you move that please?

Reply:
Done